typings/nuxt__types/index.d.ts

```diff
@@ -2,7 +2,14 @@ import '@nuxt/types'
 import type { Details as UADetails } from 'express-useragent'
 
 declare module '@nuxt/types' {
+  export interface Context {
+    $pinia: Pinia
```

Comment:
@obulat https://github.com/WordPress/openverse-frontend/pull/1230 will fix the need for this.